packages/react/src/button/Button.tsx

```diff
@@ -15,38 +15,70 @@ import { Flex } from "../flex";
 import { Text } from "../text";
 import { type Recipe, recipe } from "./Button.recipe";
 
+const presets = {
+  basic: { colorScheme: "secondary", variant: "solid" },
+  "basic-link": { colorScheme: "secondary", variant: "link" },
+  danger: { colorScheme: "danger", variant: "solid" },
+  "danger-outline": { colorScheme: "danger", variant: "outline" },
+  plain: { colorScheme: "secondary", variant: "ghost" },
+  primary: { colorScheme: "primary", variant: "solid" },
+} satisfies Record<
+  string,
+  { colorScheme: Recipe["colorScheme"]; variant: Recipe["variant"] }
+>;
+
 type ButtonProps = ExtendProps<
   ComponentPropsWithRef<"button">,
   ComponentPropsWithRef<typeof Flex>,
   {
     asChild?: boolean;
     children?: ReactNode;
+    colorScheme?: Recipe["colorScheme"];
```

Comment:
we don't need to duplicate these since we already intersected the `Recipe` type with `& Recipe`:

<img width="378" alt="Screenshot 2024-05-29 at 2 20 00 PM" src="https://github.com/optimizely-axiom/optiaxiom/assets/758244/2847bb3a-a2bb-4b01-ac14-6d3a9ee763c0">
```suggestion
```